components/shared/components/Widget/components/Widget.tsx

```diff
@@ -30,9 +42,10 @@ export const Widget = withStaticProps(async ({ preview }: { preview: boolean })
   const result = await getClient(preview).fetch<QueryResult>(widgetQuery);
 
   return {
-    text: result.widget[0],
+    widget: result.widget[0],
+    vipps: result.vipps?.[0],
   };
-})(({ text }) => {
+})(({ widget, vipps }) => {
```

Comment:
Instead of adding each payment method individually, could we take in configuration for an arbitrary amount of payment methods as an array?

Something like `paymentmethodConfigurations` or something similar (maybe shorter).